packages/application-generic/src/services/cache/key-builders/entities.ts

```diff
@@ -79,11 +80,26 @@ const buildKeyById = ({
 }): string =>
   prefixWrapper(`${type}:${keyEntity}:${identifierPrefix}=${identifier}`);
 
+const buildGroupedBlueprintsKey = ({
+  identifier,
+}: {
+  identifier: string;
+}): string =>
+  buildCommonKey({
+    type: CacheKeyTypeEnum.ENTITY,
+    keyEntity: CacheKeyPrefixEnum.GROUPED_BLUEPRINTS,
+    environmentIdPrefix: OrgScopePrefixEnum.ORGANIZATION_ID,
+    environmentId: process.env.BLUEPRINT_CREATOR,
```

Comment:
isn't the `BLUEPRINT_CREATOR` env the `organizationId` ?

Reply:
Sorry for the confusion, I changed it to prod environmentId (because it is more accurate to fetch from prod env), and then (following PR) I reverted it back to organizationId in order to make it more intuitive and if we will need to prod env id we will fetch it.

Reply:
You are right and sorry for the confusion, I changed it to prod environmentId (because it is more accurate to fetch from prod env), and then (following PR) I reverted it back to organizationId in order to make it more intuitive and if we will need to prod env id we will fetch it.